Dashboard/app/js/lib/components/devices/AssignmentsEditView/index.jsx

```diff
@@ -90,10 +90,28 @@ export default class AssignmentsEditView extends React.Component {
     };
   };
 
+  getNumberOfSelectedDevices = () => {
+    let selectedDevices = 0;
+
+    Object.keys(this.props.data.deviceGroups).forEach(dgId => {
```

Comment:
```suggestion
    Object.keys(deviceGroups).forEach(dgId => {
```
Since you use the variable several times maybe its worth it to make it a little less verbose?